src/Configurator/CopyFromPackageConfigurator.php

```diff
@@ -97,7 +97,7 @@ public function copyFile(string $source, string $target, bool $overwrite = false
             throw new LogicException(sprintf('File "%s" does not exist!', $source));
         }
 
-        copy($source, $target);
+        file_put_contents($target, $this->options->expandTargetDir(file_get_contents($source)));
```

Comment:
This replaces eg `%CONFIG_DIR%` in files copied from packages (same for recipes below).
Will allow replacing `config/bootstrap.php` by `%CONFIG_DIR%/bootstrap.php` in a few months when everyone will have moved to a more recent flex version.